src/14_cal.py

```diff
@@ -9,16 +9,36 @@
  - If the user doesn't specify any input, your program should 
    print the calendar for the current month. The 'datetime'
    module may be helpful for this.
+
  - If the user specifies one argument, assume they passed in a
    month and render the calendar for that month of the current year.
+
  - If the user specifies two arguments, assume they passed in
    both the month and the year. Render the calendar for that 
    month and year.
+
  - Otherwise, print a usage statement to the terminal indicating
    the format that your program expects arguments to be given.
    Then exit the program.
 """
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
+
+
+def render(month, year):
+    month = calendar.TextCalendar().formatmonth(year, month)
+    print(month)
+
+
+num_args = len(sys.argv)
```

Comment:
nice job with the argv. I ended up using `input().split()` to ask for user input, but this is good too.